draft-ietf-scone-protocol.md

```diff
@@ -168,6 +165,20 @@ point on the network path than the network element that signals throughput advic
 Therefore, endpoints need to respect the send rate constraints that are set by a
 congestion controller.
 
+Networks are not limited to using SCONE to communicate information
+about rate limiting policies.
+For example, a network element in an access network
+could provide throughput advice to guide application use of network capacity,
+in any way that is separate from any signals
+that are intended to influence congestion response.
```

Comment:
I see this is just moved test, but...

This document is describing SCONE but this reads like placing limits on what you can do if you are not SCONE. Maybe just "could provide throughput advice...through out-of-band mechanisms".

Reply:
Yes, thanks